### ISSUE TITLE
IE11: Exception raised when attempting to read 'tabIndex' of an 'svg' or 'use' node

### ISSUE THREAD

**Issue by ghost** [salesforce/lwc#1421]
### Description

In IE11, when attempting to read the `tabIndex` property of an `svg` or `use` node, an exception is raised. 

#### Steps to Reproduce
`this.template.querySelector('svg').tabIndex` (or `'use'`) throws a descriptor is null exception in
```js
var description = getPropertyDescriptor(Ctor.prototype, 'tabIndex');
return descriptor.get.call(this);
```

#### Expected Results
No error is thrown

#### Actual Results
An error is thrown (`descriptor` is `null`)

### Browsers Affected
Only IE11

### Version
- LWC: 1.0.2

**Comment by caridy** [comment]
I believe this is fixed on master (1.1), and it will be more tricky to fix in 1.0.x, we should confirm. /cc @ekashida

**Comment by nolanlawson** [comment]
IE11 is no longer supported, so it is unlikely we will fix this. Closing for now.
